nifpga/session.py

```diff
@@ -337,6 +348,16 @@ def datatype(self):
         indicator. """
         return self._datatype
 
+    def _get_read_function(self):
+        """ Sets this registers read function based on the data type from the
+        bitfile."""
+        return self.nifpga["Read%s" % self.datatype]
```

Comment:
self.nifpga and self.datatype never change during execution so these functions should always return the same value.  I feel like we could just call these once in the constructor of each class and avoid resolving them during every read or write call.